code/modules/mob/living/simple_animal/hostile/mushroom.dm

```diff
@@ -182,7 +182,7 @@
 /mob/living/simple_animal/hostile/mushroom/harvest()
 	var/counter
 	for(counter=0, counter<=powerlevel, counter++)
-		var/obj/item/reagent_containers/food/snacks/hugemushroomslice/S = new /obj/item/reagent_containers/food/snacks/hugemushroomslice(src.loc)
+		var/obj/item/food/snacks/hugemushroomslice/S = new /obj/item/food/snacks/hugemushroomslice(src.loc)
```

Comment:
do you need the second type reference here or can it be 
`var/obj/item/food/snacks/hugemushroomslice/S = new(src.loc)`?